apps/framework-cli/src/framework/python/wrappers/consumption_runner.py

```diff
@@ -278,10 +281,15 @@ def main():
     ch_client = get_client(interface=interface, host=host,
                            port=port, database=db, username=user, password=password)
 
-    print("Connecting to Temporal")
-    # Need to await on temporal client calls but this main function is sync
-    temporal_client = asyncio.run(get_temporal_client())
-
+    temporal_client = None
+    try:
+        # TODO: try to connect since it's still behind a feature flag
```

Comment:
Could also pass the feature flag around, lmk!

Reply:
Hmm, the error message is in the logs like this, but not in the terminal where they run `moose-cli dev`, so I guess it's sort of hidden :stuck_out_tongue:

```
Failed to connect to Temporal. Is the feature flag enabled? Failed client connect: Server connection error: tonic::transport::Error(Transport, ConnectError(ConnectError("tcp connect error", Os { code: 61, kind: ConnectionRefused, message: "Connection refused" })))
```

Reply:
Then I am fine with it